pootle/apps/pootle_app/management/commands/clear_stats.py

```diff
@@ -12,11 +12,42 @@
 # This must be run before importing Django.
 os.environ['DJANGO_SETTINGS_MODULE'] = 'pootle.settings'
 
-from pootle_app.management.commands import PootleCommand
+from django.core.management.base import NoArgsCommand
 
+from django_redis import get_redis_connection
+from redis import WatchError
 
-class Command(PootleCommand):
```

Comment:
My preference was not to go to the nuclear option (wipe out everything) but after checking the referenced issue it looks like I was the only one that was clearly asking for that. I am not sure which is the best way to proceed here.